spec/unit_tests/users_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'user'
+describe ".all" do
+  it "stores & shows the peeps" do
+    user = User.all
+    expect(user[0].username).to include "Bob"
+  end
+
+  describe ".create" do
```

Comment:
This is not describing user.all.create but user.create. Be careful with your describe loops.